bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/ValueCreator.java

```diff
@@ -830,10 +829,45 @@ public static BMap<BString, Object> createRecordValue(Module packageId, String r
     public static BMap<BString, Object> createReadonlyRecordValue(Module packageId, String recordTypeName,
                                                                   Map<String, Object> valueMap) {
         valueMap = RuntimeUtils.validateBMapValues(valueMap);
-        MapValueImpl<BString, Object> bmap = (MapValueImpl<BString, Object>) ValueUtils.createRecordValue(
+        MapValueImpl<BString, Object> bMapValue = (MapValueImpl<BString, Object>) ValueUtils.createRecordValue(
+                packageId, recordTypeName, valueMap);
+        bMapValue.freezeDirect();
+        return bMapValue;
+    }
+
+    /**
+     * Create a record value that populates record fields using the given package ID, record type name and
+     * a {@link BMap} of field names and associated values for fields.
+     *
+     * @param packageId      the package ID that the record type resides.
```

Comment:
Shall we re-align the tabs in the doc comment?
The same applies to the other comments as well.

Reply:
Addressed with bc568b8e63232c7927def3d91ae084d19f470bb7.